infra/checkout.bicep

```diff
@@ -29,7 +29,7 @@ module appPublisher 'br/public:app/dapr-containerapp:1.0.1' = {
   params: {
     location: location
     containerAppEnvName: containerAppEnvironmentName
-    containerAppName: '${nameseed}checkout'
+    containerAppName: '${name}checkout'
```

Comment:
Just want to confirm that this doesn't need resourceToken?

Reply:
I'll try using abbreviations and the resourceToken, but I do recall seeing an azd error when it expected a certain ACA name.

Reply:
Reverting to the simpler [Environment][Service] name notation for the Container Apps for the time being.
Have left the abbreviation-service-resourceToken name in as a comment to be addressed in a future PR.